wallet_test.go

```diff
@@ -0,0 +1,121 @@
+// Package implements a wallet, that holds bitcoin and operations: to deposit and withdraw money and get the wallet's balance
+
+package wallet
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestWallet_GetBalance(t *testing.T) {
+	tests := []struct {
+		name string
+		w    *Wallet
+		want Bitcoin
+	}{
+		{
+			name: "zero balance",
+			w:    &Wallet{balance: 0},
+			want: 0,
+		},
+		{
+			name: "positive balance",
+			w:    &Wallet{balance: 1},
+			want: 1,
+		},
+		{
+			name: "negative balance",
+			w:    &Wallet{balance: -1},
+			want: -1,
+		},
```

Comment:
add cases where balance is not a whole number: `4.2`, `1000000000.001`, `0.0000000000001`

Reply:
Done

Reply:
@CryptoSFor do that in other tests as well

Reply:
Done